database/migrations/20200701143540_create-users-001.js

```diff
@@ -0,0 +1,16 @@
+exports.up = function (knex) {
+  // The up function creates the users table
+  return knex.schema.createTable("users", (table) => {
+    table.increments(); //Creates column named id. It is our primary key that auto-increments
+    table.string("email").notNullable().unique();
+    table.string("first_name", 128).notNullable();
+    table.string("last_name", 128).notNullable();
+    table.string("password", 128).notNullable();
+    table.boolean("is_admin").notNullable();
+  });
+};
+
+exports.down = function (knex) {
+  // The down function destroys the fruits table if it exists. It allows us to move to any state of the database.
```

Comment:
This comment doesn't seem to relate with the project. 
Unrelated comments could cause confusions down the line.